uprotocol/communication/simplepublisher.py

```diff
@@ -33,11 +36,14 @@ def __init__(self, transport: UTransport):
             raise ValueError(UTransport.TRANSPORT_NOT_INSTANCE_ERROR)
         self.transport = transport
 
-    async def publish(self, topic: UUri, payload: UPayload) -> UStatus:
+    async def publish(
```

Comment:
should this implementation be using the calloptions?

Reply:
i see simplenotifier using it but not simplepublisher

Reply:
fixed